storage/watcher.go

```diff
@@ -56,6 +56,7 @@ func (ws *watcher) Watch(key []byte, prefix bool, startRev int64) CancelFunc {
 	}
 	// TODO: cancelFunc needs to be removed from the cancels when it is called.
 	ws.cancels = append(ws.cancels, c)
+	watcherGauge.Inc()
```

Comment:
@gyuho I gave this a quick test and found this is not the correct place to increase the watcher gauge. It should be inced at https://github.com/coreos/etcd/blob/master/storage/watchable_store.go#L167